tools/pipelines-tasks/common/helpers.ts

```diff
@@ -152,6 +152,7 @@ export async function installNuget(packagePath: string, outputPath: string, targ
         }
 
         await nugetHelper.copyTargetDlls(outputPath, targetNetFramework);
+        tl.rmRF('nuget.exe');
         tl.popd();
```

Comment:
I think these two should be in a `finally`

Reply:
Done